Bugs in TypeScript type definitions.

### High-level description

2 bugs in typescript type definitions.

### Steps to reproduce

Try to use vtkFullScreenRenderWindow from a typescript project.

### Detailed behavior

* `rootContainer` is not defined on IFullScreenRenderWindowInitialValues
* Param to `setRendertimemultiplier` needs a non-any type.

```
~$ yarn type
yarn run v1.22.15
$ vue-tsc -p tsconfig.check.json --noEmit
src/components/MeshViewer.vue:30:72 - error TS2345: Argument of type '{ rootContainer: HTMLElement; }' is not assignable to parameter of type 'IFullScreenRenderWindowInitialValues'.
  Object literal may only specify known properties, and 'rootContainer' does not exist in type 'IFullScreenRenderWindowInitialValues'.

30     const fullScreenRenderer = vtkFullScreenRenderWindow.newInstance({ rootContainer });
                                                                          ~~~~~~~~~~~~~

node_modules/@kitware/vtk.js/Rendering/Core/Prop.d.ts:235:29 - error TS7006: Parameter 'renderTimeMultiplier' implicitly has an 'any' type.

235     setRendertimemultiplier(renderTimeMultiplier): boolean;
                                ~~~~~~~~~~~~~~~~~~~~
```

### Expected behavior

Type checking should pass (without having to disable library type checks).

### Environment
- **vtk.js**: 23.0.1
- **OS**: n/a
- **Browser**: n/a

--- a/Sources/Rendering/Core/Prop/index.d.ts
+++ b/Sources/Rendering/Core/Prop/index.d.ts
@@ -14,7 +14,7 @@ export interface IPropInitialValues {
     estimatedRenderTime?: number;
     savedEstimatedRenderTime?: number;
     renderTimeMultiplier?: number;
-    textures?: Array<any>;
+    textures?: vtkTexture[];
 }
 
 export interface vtkProp extends vtkObject {
@@ -141,20 +141,20 @@ export interface vtkProp extends vtkObject {
 
     /**
      * 
-     * @param texture 
+     * @param {vtkTexture} texture The vtkTexture instance.
      *
      */
     hasTexture(texture: vtkTexture): boolean;
 
     /**
      * 
-     * @param texture 
+     * @param {vtkTexture} texture The vtkTexture instance.
      */
     addTexture(texture: vtkTexture): void;
 
     /**
      * 
-     * @param texture 
+     * @param {vtkTexture} texture The vtkTexture instance.
      */
     removeTexture(texture: vtkTexture): void;
 
@@ -230,9 +230,9 @@ export interface vtkProp extends vtkObject {
 
     /**
      * This is used for culling and is a number between 0 and 1. It is used to create the allocated render time value.
-     * @param renderTimeMultiplier 
+     * @param {Number} renderTimeMultiplier 
      */
-    setRendertimemultiplier(renderTimeMultiplier): boolean;
+    setRenderTimeMultiplier(renderTimeMultiplier: number): boolean;
 
     /**
      * Not Implemented yet

